### ISSUE TITLE
[BUG] Stitching seems to have % chance to work but doesn't mention this when stitching

### ISSUE THREAD

**Issue by Wintermote** [Blackstone-SS13/BLACKSTONE#346]
## Describe the bug
When stitching someone you will be told that you've stitched the wound, but the wound will still be there and the needle will still have all of its uses.

If you stitch many times over and over, you will eventually stitch a wound successfully, leading me to believe it is a % chance each time, based on skill level. If this is the case, then it would be best to be told that you fail to stitch the wound, rather than told that you did stitch it when you did not.

## Steps to Reproduce:
1. Have wound
2. Stitch wound with needle once
3. Examine needle and wound and see that both are still there despite being told that you stitched the wound
4. Repeat this until, eventually, stitching works and a usage of the needle is consumed

## Additional context
Basically just need to give the player more information here so they know what's happening. It's very confusing for new players how it functions at the moment.

**Comment by Wintermote** [comment]
This should be like 95% solved by #384, whenever that happens to be merged.

**Comment by ChungusGamer666** [comment]
hmmm no good

**Comment by Wintermote** [comment]
Closing since #384 was merged and mostly solves it
